cohere/client.py

```diff
@@ -154,8 +154,9 @@ def chat(self,
              return_chatlog: bool = False,
              return_prompt: bool = False,
              chatlog_override: List[Dict[str, str]] = None,
-             preamble_override: str = None,
-             username: str = None) -> Chat:
+             username: str = None,
```

Comment:
in #160 this is implemented as `user_name`

Reply:
should I change it here as well or is it better to just wait for #160 to be merged?